data/items/items.xml

```diff
@@ -17590,6 +17590,7 @@
 	</item>
 	<item id="9975" article="a" name="crystal table">
 		<attribute key="weight" value="2500" />
+		<attribute key="wrapTo" value="9974" />
```

Comment:
@wgrriffel I see, you may be right
some construction kits have "unwrap" feature, others don't
I don't know if it depends on ids or not